### PR TITLE
Post qt5 fixes

### PR DESCRIPTION
**Pull request by despair86**
merge after #6164

### CODE REVIEW

File: components/encumbered/vlc/Makefile [hunk 1]
```diff
@@ -19,6 +19,7 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= vlc
 COMPONENT_VERSION= 3.0.11.1
+COMPONENT_REVISION= 1
 COMPONENT_FMRI = media/vlc
 COMPONENT_CLASSIFICATION = Applications/Sound and Video
 COMPONENT_SUMMARY= Cross-platform media player and streaming server
```
Comment (AndWac) [inline components/encumbered/vlc/Makefile:25]: You seem to have missed:
PKG_CONFIG_PATH+=:/usr/lib/qt/5.8/lib/$(MACH64)/pkgconfig

Comment (AndWac) [reply components/encumbered/vlc/Makefile:25]: and later also
QT5_ROOT = /usr/lib/qt/5.8

**Comment by despair86** [comment]
ah I probably squished it away when I branched off

**Comment by AndWac** [comment]
Something seems to be wrong on my machine:
checking for c99... no
configure: error: Cannot find native C99 compiler: please define BUILDCC.
gmake: *** [/oibuild/oi-userland/make-rules/configure.mk:175: /oibuild/oi-userland/components/encumbered/vlc/build/amd64/.configured] Error 1

I'll check on another...

**Comment by AndWac** [comment]
...and my 2nd build machine fails with the same error. But this doesn't have the new qt version yet. Something else is wrong.
Does vlc compile on your machine?

**Comment by despair86** [comment]
yeah i added `BUILDCC=$(CC)` to `CONFIGURE_OPTIONS` but i have a conflicting X265 in my own PATH

**Comment by AndWac** [comment]
I have fixed the vlc issues with #6199 and #6200 but you'll have to re-sync and increment COMPONENT_REVISION again (plus the other fixes).

**Review by AndWac — APPROVED** [review]

**Pull request merged**